WalletWasabi.Fluent/ViewModels/SearchBar/SearchItems/IActionableItem.cs

```diff
@@ -4,5 +4,5 @@ namespace WalletWasabi.Fluent.ViewModels.SearchBar.SearchItems;
 
 public interface IActionableItem : ISearchItem
 {
-	Func<Task> OnExecution { get; }
-}
\ No newline at end of file
+	Func<Task>? OnExecution { get; }
```

Comment:
This shouldn't be nullable.

`SearchBarDesignViewModel` is only used by the designer. To show something on the IDE's preview.

Create a dummy method locally inside of the `SearchBarDesignViewModel`'s constructor and pass that to the `ActionableItem`.

Example:
```
Task DoNothing() => Task.Run(() => { });
```